timeless/message/property_resource.py

```diff
@@ -1,7 +1,13 @@
 """ Message resource for internationalization based on a property file """
 
+from configparser import ConfigParser
+
 from timeless.message.message_resource import MessageResource
 
+import os
+
+from main import app
+
 
 class PropertyResource(MessageResource):
```

Comment:
@paulodamaso Is it ``FileResource`` or ``PropertyResource``? I think you god the PR's title wrong, it should be ``Implemented PropertyResource.get(...)``

Reply:
@amihaiemil Referencs to `FileResource` corrected